tests/storage/migration.go

```diff
@@ -131,6 +132,7 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 				libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
 				libdv.WithStorage(libdv.StorageWithStorageClass(sc),
 					libdv.StorageWithVolumeSize(size),
+					libdv.StorageWithFilesystemVolumeMode(),
```

Comment:
Maybe change the signature to createFilesystemDV?

Reply:
So there is another function called StorageWithBlockVolumeMode, I just used the same pattern in the name. I could have also using StorageWithAccessMode and passing the access mode (which is what the new function does)

Reply:
Oh okay, yeah that is an option. But since this is a test suite scoped function I was not too worried about the name.